packages/internal/src/generate/helpers.js

```diff
@@ -0,0 +1,13 @@
+import path from 'path'
+
+import { getProject } from '@redwoodjs/structure'
+
+const project = getProject()
+
+export function generateTypeDef(filename, templatePath) {
+  const { host } = project
+  host.writeFileSync(
+    path.join(host.paths.types, filename),
+    project.host.readFileSync(templatePath)
+  )
+}
```

Comment:
I don't think this file is used?

Reply:
I'll delete it in the next PR! Thanks

Reply:
@dac09 @peterp Thanks, gents! I could have been more clear --> yesterday I merged after a review so that I could start QA. BUT I intended for time to revisit these items today (exactly as you're doing now), assuming we would resume work in a PR part 2. 

Thanks for doing so!

Reply:
Also, to make sure I'm tracking, my assumption is that this work is resuming over here: https://github.com/redwoodjs/redwood/pull/2710

Let me know otherwise.